src/pseudopeople/utilities.py

```diff
@@ -198,30 +198,35 @@ def cleanse_integer_columns(column: pd.Series) -> pd.Series:
 
 
 @cache
-def load_ocr_errors_dict():
+def load_ocr_errors():
```

Comment:
How does cache get reset here? Should we be worried about users having a cache with older data if we were to update ocr errors or similar data?

Reply:
Good question. The cache is in memory, so it only lives as long as the Python process. If the user installed a new version of pseudopeople, they'd need to restart Python before they would see the effect. This is already true of the code itself, so you could argue that this behavior is _better_ than the old behavior (though it still doesn't enforce that the code and data are always in lockstep).